crates/re_log_types/src/data_table.rs

```diff
@@ -1266,6 +1271,8 @@ impl DataTable {
 /// Crafts a simple but interesting [`DataTable`].
 #[cfg(not(target_arch = "wasm32"))]
 impl DataTable {
+    /// NOTE: because everything here is predetermined and there is no input we assume it's safe here
+    #[allow(clippy::unwrap_used)]
     pub fn example(timeless: bool) -> Self {
```

Comment:
// nit: since the attribute doesn't affect the interface but only the inside
```suggestion
    pub fn example(timeless: bool) -> Self {
        /// NOTE: because everything here is predetermined and there is no input we assume it's safe here
        #![allow(clippy::unwrap_used)]
```